### ISSUE TITLE
API Setting

### ISSUE THREAD

**Issue by natekfl** [Ajedi32/echovr_api_docs#6]
In the first paragraph, it says "This API is always on and does not require any options to enable it". However, at least on the quest version, the API is disabled by default and must be enabled in settings.

**Comment by Ajedi32** [comment]
Yeah, they changed that a while back. Feel free to submit a PR to add instructions for enabling and interacting with the API on Quest. (I think desktop uses a similar setting too now but not sure.)
